pkg/verification/service/verification_service_test.go

```diff
@@ -320,6 +322,71 @@ func (s *TestVerificationServiceSuite) TestInitVerificationFailsDailyCounterExce
 	require.Empty(s.T(), userSignup.Annotations[v1alpha1.UserSignupVerificationCodeAnnotationKey])
 }
 
+func (s *TestVerificationServiceSuite) TestInitVerificationFailsWhenPhoneNumberInUse() {
```

Comment:
Let's also add a test for the case when there is a deactivated usersignup with the same phone.

Reply:
Done